snuba/cli/bulk_load.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+import click
+
+from snuba import settings
+from snuba.datasets.factory import get_dataset
+from snuba.snapshots.postgres_snapshot import PostgresSnapshot
+
+
+@click.command()
+@click.option('--dataset', type=click.Choice(['events', 'groupedmessage', 'outcomes']),
+              help='The dataset to bulk load')
+@click.option('--source',
+              help='Source of the dump. Depending on the dataset it may have different meaning.')
+@click.option('--dest-table',
+              help='Clickhouse destination table.')
+@click.option('--log-level', default=settings.LOG_LEVEL, help='Logging level to use.')
+def bulk_load(dataset, dest_table, source, log_level):
+    import sentry_sdk
+
+    sentry_sdk.init(dsn=settings.SENTRY_DSN)
+    logging.basicConfig(level=getattr(logging, log_level.upper()), format='%(asctime)s %(message)s')
```

Comment:
We should refactor this later so that the SDK doesn't need to be installed in each cli script independently.